x-pack/plugins/security_solution/public/network/components/embeddables/embedded_map.tsx

```diff
@@ -245,6 +245,28 @@ export const EmbeddedMapComponent = ({
     [storage]
   );
 
+  const content = useMemo(
+    () =>
+      storageValue ? (
```

Comment:
nit: maybe we could use some if/else here so we don't have nested ternaries 
https://github.com/elastic/kibana/blob/main/STYLEGUIDE.mdx#only-use-ternary-operators-for-small-simple-code